change/@ni-nimble-components-905600e9-5f08-47e1-8b5f-345e25a2ccbc.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This reveals a potentially problematic behavior where the dropdown can appear really small due to limited space and then scrolling the page keeps it small despite more space being available. This isn't a new problem and it exists for select (and probably other dropdowns) already. 

I think the fix is #1451 where we have to do something like dismiss these popups when the page or container scrolls. I don't think we need to do anything about it in this PR, I'm mostly just raising awareness and establishing a link to that work item.

If we wanted to do something in this PR, maybe we should set a min-height of at least one item? Or open the dropdown above if there's more space there? But those would probably just be band-aid fixes so probably not worth doing given that we'd have to apply them across all components.

https://github.com/ni/nimble/assets/10500124/5684daa0-1620-46bb-8160-e10b4e64fccb

Reply:
Either way I don't think it needs to block this PR, fyi @m-akinc